packages/components/src/utils/testing/snapshot-input-testing.tsx

```diff
@@ -0,0 +1,68 @@
+import { executeSnapshotTests } from './snapshot-testing';
+
+export function executeInputSnapshotTests<Props extends Record<string, unknown>>(
```

Comment:
Ich finde die Implementierung und die Test-Szenarien grundsätzlich schon sehr gut.

In dieser Funktion haben wir einen kleinen "Stilbruch":
`executeSnapshotTests` kann man mit einem Array an Prop-Varianten aufrufen. `executeInputSnapshotTests` ist von Namen und Parametern her sehr ähnlich, hier brauchen wir aber mehrere Aufrufe, um verschiedene Prop-Varianten testen zu können. Wollen wir das noch anpassen, so dass hier auch ein Array unterstützt wird?

Reply:
Wir haben eigentlich keinen Stillbruch, auch wenn diese Methoden ähnlich sind, wird die executeInputSnapshotTests nur einmal verwendet und für spezielle Dinge kann man dann wieder auf die  executeSnapshotTests zurückgreifen. Des Wegen hab ich bewusst auf Array verzichtet. Auch um die Sache nicht noch komplizierter zu machen.